company-profiles/retail-zipline.md

```diff
@@ -0,0 +1,35 @@
+# Retail Zipline
+
+## Company blurb
+
+Retail Zipline is the communications backbone for retail operations. Our customers include some of the world's most loved retailers and we are on a mission to improve the working lives of retail employees all over the world.
+
+## Company size
+
+Sub 50
+
+## Remote status
+
+Our co-founders are located in San Francisco and Kansas City. We have been 100% remote since the beginning.
+
+We communicate through Basecamp, Slack, GitHub, and Zoom.
+
+## Region
+
+Worldwide, but we are currently primarily North America. We are expanding for worldwide coverage now.
```

Comment:
Same here, most companies use this field to say a bit about where their employees currently work from or where the company can hire from.